matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/mapper/RoomSummaryMapper.kt

```diff
@@ -88,7 +88,9 @@ internal class RoomSummaryMapper @Inject constructor(private val timelineEventMa
                             order = it.order,
                             autoJoin = it.autoJoin ?: false,
                             viaServers = it.viaServers.toList(),
-                            parentRoomId = roomSummaryEntity.roomId
+                            parentRoomId = roomSummaryEntity.roomId,
+                            // What to do here?
```

Comment:
Why this question?

Reply:
Forgot to remove the comment (suggested has been added to database)